src/App.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import './App.scss';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/todoList';
 
-// import todos from './api/todos';
-// import users from './api/users';
+const preparedTodos = todos.map(todo => {
+  const user = users.find(person => person.id === todo.userId) || null;
+
+  return { ...todo, user };
+});
```

Comment:
можна скоротити код, якщо з map зразу повертати об'єкт
```suggestion
const preparedTodos = todos.map(todo => ({
  ...todos,
  user: users.find(person => person.id === todo.userId) || null,
}));
```